components/Bounty/BountyModal.tsx

```diff
@@ -437,7 +437,7 @@ function BountyModal({
 const bountyTooltip = StyleSheet.create({
   tooltipContainer: {
     textAlign: "left",
-    width: 450,
+    width: 300,
```

Comment:
The tooltip overflow the modal and was hidden as a result. Reducing the width resolved this.
<img width="683" alt="Screenshot 2023-12-13 at 9 14 48 AM" src="https://github.com/ResearchHub/researchhub-web/assets/16143968/8e322a32-66d2-4af1-921b-ac9e1f86ff89">